Tests/PrivateNearestNeighborSearchTests/CiphertextMatrixTests.swift

```diff
@@ -32,12 +32,14 @@ func randomData<T: ScalarType>(dimensions: MatrixDimensions, modulus: T) -> [[T]
     }
 }
 
-final class CiphertextMatrixTests: XCTestCase {
-    func testEncryptDecryptRoundTrip() throws {
+@Suite("Ciphertext Matrix Tests")
+struct CiphertextMatrixTests {
+    @Test("Encrypt/Decrypt Round Trip")
```

Comment:
Similarly, let's just use
```suggestion
    @Test
```
That'll be easier to migrate and keep in sync with the test.